pages.de/android/screencap.md/screencap.md

```diff
@@ -0,0 +1,9 @@
+# Bildschirmfotos
+
+> Nimmt ein Bildschirmfoto von einem Handy-Display auf
+> Dieser Befehl kann nur über die 'adb shell' benutzt werden
+> Für weitere Informationen: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Ein Bildschirmfoto aufnehmen:
+
+`screencap {{path/to/file}}`
```

Comment:
```suggestion
`screencap {{pfad/zu/datei}}`
```
We translate the path placeholders across our pages, allowing native readers to better understand the examples.